contrib/mesos/pkg/executor/service/podsource.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
right, we talked about this. i wasn't crazy about moving this into its own package simply for type private/public checking -- i'm not expecting anyone to abuse this, and it's ugly hack code which is why I think it belongs in /service